Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java

```diff
@@ -40,6 +42,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         final City mCity = (City) getIntent().getSerializableExtra(EXTRA_MESSAGE_CITY_OBJECT);
+        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
```

Comment:
This change can cause app to crash: see this: https://github.com/project-travel-mate/Travel-Mate/pull/735/files

Reply:
I can see you fixed that . So whats the issue?

Reply:
that has been fixed for FunFactsActivity.java 
Do similar thing here also